Code/Yahoo.Yui.Compressor.Tests/JavaScriptCompressorTaskTests.cs

```diff
@@ -182,6 +182,23 @@ public void An_Error_Is_Not_Logged_If_The_Output_File_Has_The_Same_Name_But_A_Di
             Assert.That(BuildEngine.DoesNotContainError(compressor.BuildEngine, ("Output file cannot be the same as source file(s).")));
         }
 
+        [Test]
+        public void If_OutputFile_Directory_Does_Not_Exist_It_Is_Created()
+        {
+            // Arange
+            var compressor = CreateCompressorTask();
+            compressor.SourceFiles = new ITaskItem[] { new TaskItem("Javascript Files\\SampleJavaScript1.js") };
+            compressor.OutputFile = "NewOutputDir\\Output.js";
+            if (Directory.Exists(Path.GetFullPath("NewOutputDir")))
+                Directory.Delete(Path.GetFullPath("NewOutputDir"), true);

```

Comment:
Based on [Directory.Delete()](https://msdn.microsoft.com/en-us/library/fxeahc5f(v=vs.110).aspx)'s documentation I don't see a reason for it to fail silently. It is a fairly simple method call which throws an exception when it is unable to delete the directory which should fail the test case.